spec/inventory_spec.rb

```diff
@@ -17,4 +18,40 @@
     expect(obj.id).to   be_a Integer
     expect(obj.name).to be_a String
   end
+
+  describe "#inventory_sources" do
+    it "#inventory_sources returns a list of inventory_sources" do
+      expect(api).to receive(:get).and_return(instance_double("Faraday::Result", :body => inventory_source.to_json))
+      inventory_sources = described_class.new(api, raw_instance).inventory_sources
+      expect(inventory_sources).to be_a Array
+      expect(inventory_sources.count).to eq 1
+    end
+  end
+
+  describe "#update_all_inventory_sources" do
+    let(:can_update_true)   { {'can_update' => true}  }
+    let(:can_update_false)  { {'can_update' => false} }
+    let(:post_result_body) { {:inventory_update => 1} }
+    let(:inventory_sources) do
```

Comment:
I guess I was confused because it is plural, but returns a single object.  it then gets wrapped in an array [here](https://github.com/ManageIQ/ansible_tower_client/pull/31/files#diff-7fedb3c455356ad23796abdd4747cd91R41).  I guess I was expecting it to be singular or return an array.